### PR TITLE
test: Use nanoid instead of UUID in e2e tests for wf names (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Contains 2 changes to e2e test:
- test: Enable running only single suite with test:e2e:all script. E.g. `pnpm test:e2e:all 1-workflows.cy`
- test: Use nanoid instead of uuid when generating a random workflow name

We have been adding a random uuid to workflow names in some e2e tests.
Cypress is actually pretty slow in typing text to input fields. Since
uuid is 36 characters, this adds some delay to our e2e tests. This
change shaves some seconds off by using a 12 character long
nanoid instead.

On my local machine 19-execution suite execution time went from 47.3s
to 42.7s (3 test run avg). 1-workflows went from 27.3s to 24.7s.

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -36,6 +37,15 @@ Cypress.Commands.add('createFixtureWorkflow', (fixtureKey, workflowName) => {
 	workflowPage.actions.zoomToFit();
 });
 
+Cypress.Commands.add(
+	'createFixtureWorkflowWithUniqueName',
```
Comment (netroy) [inline cypress/support/commands.ts:41]: what's the advantage of creating a new command vs updating `createFixtureWorkflow` to use a unique name for every workflow?

Comment (tomi) [reply cypress/support/commands.ts:41]: Yeah I think it might be clearer to just one have command

**Review by netroy — APPROVED** [review]
🎉

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Pull request merged**